Docs: Setup OAuth providers using the SSO Settings UI (#81589)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

## Before you begin

To follow this guide, ensure you have permissions in your Okta workspace to create an OIDC app.

## Configure Okta authentication client using the Grafana UI

{{% admonition type="note" %}}
Available in Public Preview in Grafana 10.4 behind the `ssoSettingsApi` feature toggle.
{{% /admonition %}}

As a Grafana Admin, you can configure generic OAuth2 client from within Grafana using the Okta UI. To do this, navigate to **Administration > Authentication > Okta** page and fill in the form. If you have a current configuration in the Grafana configuration file then the form will be pre-populated with those values otherwise the form will contain default values.

> After you have filled in the form, click **Save** to save the configuration. If the save was successful, Grafana will apply the new configurations.

Reviewer: ```suggestion
After you have filled in the form, click **Save**. If the save was successful, Grafana will apply the new configurations.
```

> In case you would like to reset your changes you made on the UI back to the default values, click **Reset**. After you have reset the changes, Grafana will apply the configuration from the Grafana configuration file (if there is any configuration) or the default values.

Reviewer: ```suggestion
If you need to reset changes you made in the UI back to the default values, click **Reset**. After you have reset the changes, Grafana will apply the configuration from the Grafana configuration file (if there is any configuration) or the default values.
```
